sql/moz-fx-data-shared-prod/firefox_accounts/nonprod_fxa_content_auth_stdout_events/view.sql

```diff
@@ -1,7 +1,7 @@
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.nonprod_fxa_content_auth_stdout_events`
 AS
-  --
+  -- TODO: we should look into deleting this view.
```

Comment:
Please add some context/links for the next person reading this comment.

Reply:
ok, added a bit more context to this comment. I hope that adds clarity.